.github/ISSUE_TEMPLATE/release.md

```diff
@@ -14,9 +14,10 @@ labels: ["release"]
 ### At the beginning of the cycle
 
 - [ ] Obtain the Release coordinator's contact from the `@k6-browser` team and co-assign the issue to them.
-- [ ] Create a new `release-v0.5x.0` long-lived branch and add a new release notes file using the available [template](https://github.com/grafana/k6/blob/master/release%20notes/template.md) to the [repository's `release notes` folder](https://github.com/grafana/k6/blob/master/release%20notes).
+- [ ] Open and merge a pull-request for bumping [the k6 Go project's version](https://github.com/grafana/k6/blob/master/lib/consts/consts.go#L11-L12).
```

Comment:
In this way, the build on master is more representative of the correct version. It can be a preview of the next release.
We don't plan them but if you think of building nightly then the current approach would generate the wrong version.

Reply:
Plus, it isn't required to have them at the end, so isn't better to reduce the steps we have to do on the release day?

Reply:
In the case of the nightly version, I believe the more important version is not the constant but the build info since we need to explicitly mention their day/commit instead of just the next minor/major version.

> Plus, it isn't required to have them at the end, so isn't better to reduce the steps we have to do on the release day?

Since this is a very straightforward action, and even now, almost all such PR is approved at the bugging of the cycle, I don't see an issue in just pressing the button on the very last day.

Reply:
>I don't see an issue in just pressing the button on the very last day.

In that case, it means we are going to have the pull request open for the entire cycle.

Btw, this is not relevant to the current proposal. I thought it was a small improvement but we can revert it.

Reply:
Done